cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/logging/Log.java

```diff
@@ -42,6 +42,13 @@
         System.setProperty("user.language", "en");
         Locale.setDefault(new Locale("en", "EN"));
 
+        // change global stdout log format
+        // desired output format: <date> <time> <logger name> [<loglevel>]: <message>
+        System.setProperty(
+                "java.util.logging.SimpleFormatter.format",
+                "%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS %3$s [%4$-5s] %5$s%6$s%n"
```

Comment:
The string values could be stored in constant fields, but I don't know if that is really beneficial in this case as they probably won't be used elsewhere.

Reply:
Those are really not used anywhere else. If you need to change them, it's just as easy to do it directly there.

It's already pretty bad practice to set properties this way, but well...